python/pyenv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    pyenv
+version                 2.3.2
```

Comment:
use the `github.setup` syntax instead of specifying `name` and `version` separately

Reply:
and use `github.tarball_from archive'